src/main.py

```diff
@@ -49,7 +48,7 @@
 
 
 app.include_router(gsp_router, prefix=f"{v0_route}/gsp")
-# app.include_router(pv_router, prefix=f"{v0_route}/pv")
+app.include_router(pv_router, prefix=f"{v0_route}/pv")
```

Comment:
As discussed with @JackKelly on Slack previously: We should **never** be exposing this data, no matter if it's behind authentication or not. We're not the data owners and neither do we have the right to publish it, according to our license agreement.

While the authentication is nice, as it won't make the data publicly available, we are **not allowed** to redistribute this _raw_ data to our customers in any way.